src/main/resources/io/jenkins/plugins/checks/github/status/GitHubSCMSourceStatusChecksTrait/config.jelly

```diff
@@ -1,13 +1,7 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:i="/io/jenkins/plugins/checks/github/status">
```

Comment:
this is the convention normally used
```suggestion
<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:g="/lib/github-checks/status">
```

Reply:
you mean I'll need to create new folders specifically to include these lib files?